arabic/learn/whattolearn/life/life.md

```diff
@@ -0,0 +1 @@
+انا يتيم الاب فان نظرتي للحياة مختلفة و مشوقة فلانني بدون معلم فاخترت ان اتعلم من ابطال عاشو قبلي ,فانني لن ارضخ لجهل مهما كانت الظروف كتاب في جيبي معي اينما ذهبت رافقني .
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.